src/formElementPlugins/dropdownChoice/localisation/default/strings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ADD_CUSTOM_OPTION_LABEL": "Add custom value: "
```

Comment:
The "Add " text that comes up when adding a custom option needs to be localised. Follow the same pattern as localised strings in other form elements.